src/kibana/components/agg_response/geo_json/rowsToFeatures.js

```diff
@@ -16,30 +16,42 @@ define(function (require) {
       var geohash = unwrap(row[geoI]);
       if (!geohash) return;
 
+      // fetch latLn of northwest and southeast corners, and center point
       var location = decodeGeoHash(geohash);
-      var center = [
+
+      var centerLatLng = [
+        location.latitude[2],
+        location.longitude[2]
+      ];
+
+      // geoJson coords use LngLat coordinates
+      // http://geojson.org/geojson-spec.html#positions
+      // "longitude, latitude, altitude for coordinates in a geographic coordinate reference system"
+      var centerLngLat = [
         location.longitude[2],
         location.latitude[2]
       ];
```

Comment:
Would it make sense to do something like `var centerLngLat = centerLatLng.slice().reverse();` instead?